homeassistant/components/openai_conversation/__init__.py

```diff
@@ -361,4 +361,34 @@ async def async_migrate_integration(hass: HomeAssistant) -> None:
                 title=DEFAULT_NAME,
                 options={},
                 version=2,
+                minor_version=2,
             )
+
+
+async def async_migrate_entry(hass: HomeAssistant, entry: OpenAIConfigEntry) -> bool:
+    """Migrate entry."""
+    LOGGER.debug("Migrating from version %s:%s", entry.version, entry.minor_version)
+
+    if entry.version > 2:
+        # This means the user has downgraded from a future version
+        return False
+
+    if entry.version == 2 and entry.minor_version == 1:
```

Comment:
Isn't the default minor version 0 ?

Reply:
No

https://github.com/home-assistant/core/blob/0a6608049bf2d8b9dd40c7ca4fd487dc59ca0109/homeassistant/data_entry_flow.py#L635-L637